docs/neofs-blockstorage.md

```diff
@@ -102,33 +92,24 @@ shuts down automatically.
 
 ### NeoFS block uploading command
 The `util upload-bin` command is designed to fetch blocks from the RPC node and upload 
-them to the NeoFS container. It also creates and uploads index files. Below is an
-example usage of the command:
+them to the NeoFS container. Below is an example usage of the command:
```

Comment:
Something should be told about batching, because it's important to use this mechanism correctly.